FibonacciNumbersWithGenerators.py

```diff
@@ -1,18 +1,31 @@
-def fibonacci_generator(n = None):
+def fibonacci_generator(n=None):
     """
-        Generating function up to n fibonacci numbers iteratively
-        Params:
-            n: int
-        Return:
-            int
+    Generating function up to n fibonacci numbers iteratively
+    Params:
+        n: int
+    Return:
+        int
+    """
+
+
+def fibonacci_generator(n=None):
+    """
+    Generating function for up to n Fibonacci numbers iteratively.
+    Params:
+        n: int
+    Return:
+        int
     """
     f0, f1 = 0, 1
     yield f1
-    while n == None or n > 1:
+    while n is None or n > 1:
```

Comment:
checking `None` is unnecessary for more readable code I'd something like 
```python
while not n or n>1
```
or perhaps 
```python
# if n is None loop terminates auto.
while n>1
```